src/components/shared/Button.tsx

```diff
@@ -12,28 +12,53 @@ import styled from 'styled-components/native';
 
 import { ratio, colors } from '../../utils/Styles';
 
+interface IStyledElement {
+  white?: boolean;
+};
+
+interface IButtonContainer {
+  width: number;
+  height: number;
+};
+
+interface IProps {
+  testID: string;
+  containerStyle: ViewStyle;
+  isWhite: boolean;
+  isLoading: boolean;
+  isDisabled: boolean;
+  onPress: () => void;
+  imgLeftSrc: any;
+  indicatorColor: string;
+  activeOpacity: number;
+  children: any;
+  width: number;
+  height: number;
+}
+
 const StyledContainer = styled.View`
   flex: 1;
 `;
 
-const StyledButton = styled.View`
-  background-color: ${colors.dodgerBlue};
+const StyledButtonContainer = styled.View<IButtonContainer>`
+  width: ${({ width }) => width};
+  height: ${({ height }) => height};
+  justify-content: center;
+  align-items: center;
+`;
+
+const StyledButton = styled.View<IStyledElement>`
```

Comment:
Same here. Since `typing1` in `stlyed-component` is bad currently, I didn't planned to type the `styles` but if there is good solution, I'd prefer too.

I am still confused with this styling. Hope you can teach me.

Reply:
I just did them to avoid lint errors, the how-to is pretty much [here](https://www.styled-components.com/docs/api#using-custom-props)

Reply:
@marsinearth Thanks for the detail!